crypto/tmhash/hash.go

```diff
@@ -58,8 +58,9 @@ func NewTruncated() hash.Hash {
 	}
 }
 
-// SumTruncated returns the first 20 bytes of SHA256 of the bz.
+// SumTruncated returns the first 32 bytes of SHA256 of the bz.
+// NOTE: friday changes the length of all addresses to 32 bytes.
 func SumTruncated(bz []byte) []byte {
 	hash := sha256.Sum256(bz)
```

Comment:
sha256?

Reply:
it's for ed25519 or bls. not used secp256k1

Reply:
Confirmed. SumTruncated -> AddressHash -> threashold_pubkey 
SumTruncated is only used in BLS and ED25519